sdk/eventgrid/Azure.Messaging.EventGrid/api/Azure.Messaging.EventGrid.netstandard2.0.cs

```diff
@@ -1,9 +1,15 @@
 namespace Azure.Messaging.EventGrid
 {
+    public static partial class BinaryDataExtensions
+    {
+        public static Azure.Messaging.EventGrid.CloudEvent ToCloudEvent(this Azure.Core.BinaryData binaryData) { throw null; }
+        public static Azure.Messaging.EventGrid.EventGridEvent ToEventGridEvent(this Azure.Core.BinaryData binaryData) { throw null; }
```

Comment:
Are we sure we want to start shipping extension methods for an exchange type like this?

Reply:
@KrzysztofCwalina suggested adding this

Reply:
VS shows extension methods from all referenced packages ignoring extensions.

![image](https://user-images.githubusercontent.com/1697911/91108788-c8807500-e62d-11ea-8a5e-d00f1e961e27.png)

Do we really want to make `BinaryData` an extension method dumping ground?

Reply:
I guess something else to consider is that even when thinking strictly from service bus context, not every message will be convertible to an eg event. TryParse might be a better pattern here.

Reply:
I guess something else to consider is that even when thinking strictly from service bus context, not every message will be convertible to an eg event. TryParse might be a better pattern here.